File: generated-sources/java-vertx/mojang-api/src/main/java/org/openapitools/server/api/verticle/SecurityQuestionAnswerApi.java
package org.openapitools.server.api.verticle;

import org.openapitools.server.api.model.Error;
import org.openapitools.server.api.MainApiException;
import org.openapitools.server.api.model.SecurityAnswer;
import org.openapitools.server.api.model.SecurityChallenge;

import io.vertx.core.AsyncResult;
import io.vertx.core.Handler;

import java.util.List;
import java.util.Map;

public interface SecurityQuestionAnswerApi  {
    //checkSecurityStatus
    void checkSecurityStatus(Handler<AsyncResult<Void>> handler);
    
    //listPendingSecurityQuestions
    void listPendingSecurityQuestions(Handler<AsyncResult<List<SecurityChallenge>>> handler);
    
    //sendSecurityQuestionAnswers
    void sendSecurityQuestionAnswers(List<SecurityAnswer> securityAnswer, Handler<AsyncResult<Void>> handler);
    
}

File: generated-sources/dart-jaguar/mojang-api/lib/api/security_question_answer_api.dart
import 'package:jaguar_retrofit/annotations/annotations.dart';
import 'package:jaguar_retrofit/jaguar_retrofit.dart';
import 'package:jaguar_serializer/jaguar_serializer.dart';
import 'package:jaguar_serializer/src/repo/repo.dart';
import 'dart:async';

import 'package:openapi/model/security_answer.dart';
import 'package:openapi/model/security_challenge.dart';
import 'package:openapi/model/error.dart';


part 'security_question_answer_api.jretro.dart';

@GenApiClient()
class SecurityQuestionAnswerApi extends _$SecurityQuestionAnswerApiClient implements ApiClient {
    final Route base;
    final SerializerRepo serializers;

    SecurityQuestionAnswerApi({this.base, this.serializers});

    /// Check if security questions are needed
    ///
    /// 
    @GetReq(path: "/user/security/location", metadata: {"auth": [ {"type": "http", "name": "PlayerAccessToken" }]})
    Future<void> checkSecurityStatus(
    );

    /// Get list of questions
    ///
    /// 
    @GetReq(path: "/user/security/challenges", metadata: {"auth": [ {"type": "http", "name": "PlayerAccessToken" }]})
    Future<List<SecurityChallenge>> listPendingSecurityQuestions(
    );

    /// Send back the answers
    ///
    /// 
    @PostReq(path: "/user/security/location", metadata: {"auth": [ {"type": "http", "name": "PlayerAccessToken" }]})
    Future<void> sendSecurityQuestionAnswers(
        
        @AsJson() List<SecurityAnswer> securityAnswer
    );


}
